### PR TITLE
Ammo can imp rack

### PR DESCRIPTION
**Pull request by LynxSolstice**
# About the pull request

Adds a relatively scarce logistics backpack item to the req vendor, allows Marines to carry 3 of any obj/item/ammo_box/, increases the logistics capability of your standard Marine. (Special thanks to Sionnach for the sprites, and Bengus, Harry, and Beagle for help with the code!)

# Explain why it's good for the game

Instead of 400 boxes of ammo or mags remaining at the FOB, this new item encourages logistics and ammo-bearing (A lot of riflemen like wearing ammo boxes on their backs, this allows them to hold more boxes + MRE boxes/flare boxes as well, thanks to those being a subtype of /ammo_box/. Allows the movement of supplies easier if you can get your hands on this elusive, limited issue item!

# Testing Photographs and Procedure

image
image
image
image

# Changelog

:cl:LynxSolstice, Bengus, Sionnach, Harry, Beagle
add: Adds a new IMP framed back item, the Ammo Rack.
balance: Added the new Ammo Rack to the Req vendor. At a scaling of (round*1), it should be pretty scarce.
imageadd: Added icons for Ammo_Pack_ and onmobs for each state (0, 1, 2, and 3)
/:cl:

### CODE REVIEW

**Comment by LynxSolstice** [comment]
> image image You messed up some sprites

Fixed!

File: code/game/machinery/vending/vendor_types/requisitions.dm [hunk 1]
```diff
@@ -75,6 +75,7 @@
 		list("Mortar Shell Backpack", round(scale * 1), /obj/item/storage/backpack/marine/mortarpack, VENDOR_ITEM_REGULAR),
 		list("Technician Welder-Satchel", round(scale * 5), /obj/item/storage/backpack/marine/engineerpack/satchel, VENDOR_ITEM_REGULAR),
 		list("Small Radio Telephone Pack", round(scale * 4), /obj/item/storage/backpack/marine/satchel/rto/small, VENDOR_ITEM_REGULAR),
+		list("IMP Ammo Rack", round(scale*2), /obj/item/storage/backpack/marine/ammo_rack, VENDOR_ITEM_REGULAR),
```
Comment (morrowwolf) [inline code/game/machinery/vending/vendor_types/requisitions.dm:78]: scale*1

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/game/machinery/vending/vendor_types/requisitions.dm (hunk 1 above)
Comment (LynxSolstice) [reply code/game/machinery/vending/vendor_types/requisitions.dm:78]: You got it

**Review by nauticall — APPROVED** [review]
sprites look good!

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Make it throw range 1, add additional slowdown when carrying or wearing it.

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by cm13live** [comment]
This pull request has been mentioned on **CM-SS13**. Please provide feedback here:

https://forum.cm-ss13.com/t/three-improvements-for-marine-combat/2233/4

**Pull request closed without merging**